examples/simple/main.go

```diff
@@ -7,9 +7,15 @@ import (
 	"github.com/pion/mediadevices/examples/internal/signal"
 	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
 	_ "github.com/pion/mediadevices/pkg/codec/opus"     // This is required to register opus audio encoder
+	_ "github.com/pion/mediadevices/pkg/codec/vpx"
+	"github.com/pion/mediadevices/pkg/frame"
 	"github.com/pion/webrtc/v2"
 )
 
+const (
+	videoCodecName = webrtc.VP8
```

Comment:
Revert it to the original one, H264.

Reply:
Oh, firefox doesn't support H264.

Reply:
Oh really. I didn't know that. I was wondering why the example didn't work for H264.... Thanks